.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run Tox
+
+on:
+  push:
```

Comment:
We'd like builds to happen on every branch, not just master.

Reply:
Fixed in 3c18ad8 (next commit)

Reply:
Should the actions have run when you pushed this change? Where would the output of the actions be seen?

Reply:
You can see it running against my fork here:
https://github.com/scriptsrc/Yamale/actions

But these actions won't run on 23andMe's copy until we merge these yml files to master.